README.md

```diff
@@ -70,6 +74,15 @@ sudo update-rc.d cmdport defaults
 sudo update-rc.d -f cmdport remove
 ```
 
+Optional:
+
+Run `install\linux\register-cron.ps1`. This would register wave to start each time the system starts. Note that this step is required for service shutdown/restart automation scenario.
```

Comment:
This is already registered as a part of init script. Why do we need to add this additionally to cron?